Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM ubuntu:latest
 
-RUN sudo apt-get install -y software-properties-common && sudo add-apt-repository ppa:virtuslab/git-machete && sudo apt-get update && sudo apt-get install -y python3-git-machete 
+RUN sudo apt-get install -y software-properties-common \
+    && sudo add-apt-repository ppa:virtuslab/git-machete \
+    && sudo apt-get update \
+    && sudo apt-get install -y python3-git-machete 
```

Comment:
Okay! now let's proceed with CircleCI automation, then I'll probably have yet another comment to this Dockerfile itself